mLRS/Common/hal/hal.h

```diff
@@ -300,8 +300,9 @@ Note: Some "high-level" features are set for each device in the device_conf.h fi
 
 #if !defined FREQUENCY_BAND_2P4_GHZ && \
     !defined FREQUENCY_BAND_915_MHZ_FCC && !defined FREQUENCY_BAND_868_MHZ && \
-    !defined FREQUENCY_BAND_433_MHZ
-  #error At least one frequency band must be defined !
+    !defined FREQUENCY_BAND_433_MHZ && !defined FREQUENCY_BAND_70_CM_HAM
+  #error Correct frequency band must be defined !
+
```

Comment:
here it went a bit wrong
shoudl have stayed `#error At least one frequency band must be defined !`
and also an empty line got added

Reply:
Fixed